.rubocop.yml

```diff
@@ -23,6 +23,7 @@ Metrics/MethodLength:
     - lib/hydra/works/models/concerns/generic_file/virus_check.rb
 
 Metrics/AbcSize:
+  Enabled: false
```

Comment:
We put this in because if it's not in, rubocop fails.  It exist presently in curation concersn:

https://github.com/projecthydra-labs/curation_concerns/blob/50966373f857ddc9a04e8ef79904b00ccd73650b/.rubocop.yml#L20